client/src/annoMatrix/crossfilter.ts

```diff
@@ -30,7 +31,10 @@ function _dimensionNameFromDf(field: Field, df: Dataframe): string {
   return _dimensionName(field, colNames);
 }
 
-function _dimensionName(field: Field, colNames: string | string[]): string {
+function _dimensionName(
+  field: Field,
+  colNames: string | string[] | Int32Array
```

Comment:
This should be typed with Dataframe   `LabelType | LabelArray`

(see my comment on the definition of WhereCacheColumnLabels)

Reply:
Updated [crossfilter.ts:_dimensionName](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/crossfilter.ts#L41).